sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-sources/files/6.1/z0002-Revert-scripts-pahole-flags.sh-Parse-DWARF-and-gener.patch

```diff
@@ -1,32 +1,30 @@
-From 7246ee058019099c59f9cce29b861df0613c0236 Mon Sep 17 00:00:00 2001
-From: Jeremi Piotrowski <jpiotrowski@microsoft.com>
-Date: Wed, 10 May 2023 08:48:09 +0200
-Subject: [PATCH 2/2] Revert "scripts/pahole-flags.sh: Parse DWARF and generate
- BTF with multithreading."
+From 0e721d5a79f9c96ac886838f52d5eb5670697f6d Mon Sep 17 00:00:00 2001
+From: Sayan Chowdhury <schowdhury@microsoft.com>
+Date: Fri, 9 Feb 2024 09:35:14 +0000
+Subject: [PATCH 2/2] Revert "scripts/pahole-flags.sh: Parse DWARF and generate BTF
+ with multithreading."
 
 This breaks reproducibility of vmlinux btf output. The '--sort' arg does not
 seem to work.
 
 This reverts commit b4f72786429cc57383ff41e02407726313ca178d.
-
-Signed-off-by: Jeremi Piotrowski <jpiotrowski@microsoft.com>
 ---
  scripts/pahole-flags.sh | 3 ---
  1 file changed, 3 deletions(-)
 
 diff --git a/scripts/pahole-flags.sh b/scripts/pahole-flags.sh
-index 0d99ef17e4a5..c293941612e7 100755
+index 728d55190d97..4b1ea6c0cd9a 100755
 --- a/scripts/pahole-flags.sh
 +++ b/scripts/pahole-flags.sh
-@@ -16,8 +16,5 @@ fi
+@@ -16,9 +16,6 @@ fi
  if [ "${pahole_ver}" -ge "121" ]; then
- 	extra_paholeopt="${extra_paholeopt} --btf_gen_floats"
+        extra_paholeopt="${extra_paholeopt} --btf_gen_floats"
  fi
 -if [ "${pahole_ver}" -ge "122" ]; then
--	extra_paholeopt="${extra_paholeopt} -j"
+-       extra_paholeopt="${extra_paholeopt} -j"
 -fi
- 
- echo ${extra_paholeopt}
--- 
+ if [ "${pahole_ver}" -ge "124" ]; then
+        # see PAHOLE_HAS_LANG_EXCLUDE
+        extra_paholeopt="${extra_paholeopt} --lang_exclude=rust"
+--
 2.25.1
```

Comment:
The new patch we have also covers pahole version 125, does it make sense to use that?
https://github.com/flatcar/scripts/blob/main/sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-sources/files/6.6/z0002-revert-pahole-flags.patch

Reply:
Probably yes, because I see 1.25 in this list: https://beta.release.flatcar-linux.net/amd64-usr/current/flatcar_developer_container_packages.txt